table.rst

```diff
@@ -0,0 +1,43 @@
+
+Table for astype
+
+
++-------------------+-------------------------+
+|  S1               |  Dtype                  |
++===================+=========================+
+|  int              |  Int8Dtype,Int16Dtype,  |
+|                   |  Int32Dtype,Int54Dtype, |
++-------------------+-------------------------+
+|  str              | StringDtype,str         |
++-------------------+-------------------------+
+|  bytes            | BolleanDtype            |
++-------------------+-------------------------+
+|  float            |Float32Dtype,Float64Dtype|
```

Comment:
similar to `int`, needs `float`, `"float"`, and various numpy types.  Also should be instances of the classes you list.